src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts

```diff
@@ -446,6 +459,66 @@ class ZWaveJSConfigDashboard extends SubscribeMixin(LitElement) {
                   </p>
                 </div>
               </ha-card>
+              <ha-card>
+                <div class="card-header">
+                  <h1>
+                    ${this.hass.localize(
+                      "ui.panel.config.zwave_js.dashboard.nvm_backup.title"
+                    )}
+                  </h1>
+                </div>
```

Comment:
```suggestion
                <h1 class="card-header">
                  ${this.hass.localize(
                    "ui.panel.config.zwave_js.dashboard.nvm_backup.title"
                  )}
                </h1>
```

Or you set the `.header` prop for `ha-card`